GVFS/GVFS.UnitTests/Git/GitProcessTests.cs

```diff
@@ -14,5 +14,80 @@ public void TryKillRunningProcess_NeverRan()
             GitProcess process = new GitProcess(new MockGVFSEnlistment());
             process.TryKillRunningProcess().ShouldBeTrue();
         }
+
+        [TestCase]
+        public void ResultHasNonWarningErrors_SingleLine_AllWarnings()

```

Comment:
I realize we didn't have any before but should we have at least a couple of success tests?

Reply:
Likely so, I'll make that change.